src/ui/src/wds/WdsFieldWrapper.vue

```diff
@@ -7,22 +7,27 @@
 					>&nbsp;:&nbsp;{{ unit }}</span
 				>
 			</label>
-			<button
+			<WdsButton
 				v-if="helpButton"
 				class="WdsFieldWrapper__title__help"
-				variant="subtle"
-				:title="typeof helpButton === 'string' ? helpButton : undefined"
+				variant="neutral"
+				size="smallIcon"
```

Comment:
@ramedina86 , about the help button, the issue is: The help icon includes the circle ([here](https://fonts.google.com/icons?selected=Material+Symbols+Outlined:help:FILL@0;wght@400;GRAD@0;opsz@24&icon.query=help&icon.size=24&icon.color=%23e8eaed)), thus `tertiary` variant feels weird

![image](https://github.com/user-attachments/assets/ad5af4f9-a275-4d94-a429-882b9dd0dc35)

I thought about using `question_mark` icon instead, but it's still not that pretty

![image](https://github.com/user-attachments/assets/6f9b461f-5cbf-420d-8a72-3f49ba3bbab0)


So I decided to stik to `neutral` and this is the result.

https://github.com/user-attachments/assets/4d42933e-c4f8-4fd7-9d59-0735f96e3650

WDYT ?